src/lib/routes/controller.ts

```diff
@@ -114,6 +125,8 @@ export default class Controller {
             this.useContentTypeMiddleware(options),
             this.useRouteErrorHandler(options.handler.bind(this)),
         );
+
+        this.app.use(options.path, checkOpenAPIValidationError());
```

Comment:
```suggestion
        this.app.use(options.path, checkOpenAPIValidationError);
```
Based on my previous comment, now just use the constant (we could rename the constant to something more meaningful like `openapiErrorValidationMiddleware` or something similar)

Reply:
@gastonfournier 

I changed it as the constant on b4702e2 commit and renamed the function on 0b71d03 commit.